docker/db.py

```diff
@@ -65,25 +70,25 @@ def pg_version_check():
 def drop_pgosm_db():
     """Drops the pgosm database if it exists."""
     sql_raw = 'DROP DATABASE IF EXISTS pgosm;'
-
     conn = get_db_conn(db_name='postgres')
-    cur = conn.cursor()
-    # Required to drop DB
-    conn.set_isolation_level(0)
```

Comment:
psycopg2 also has constant values for isolation level. but having autocommit as isolevel was just a mistake made 15 years ago :smile: https://www.psycopg.org/docs/extensions.html#isolation-level-constants